package.json

```diff
@@ -5,6 +5,7 @@
   "description": "JS dependencies for Exonum docs",
   "dependencies": {},
   "devDependencies": {
+    "broken-link-checker": "^0.7.8",
```

Comment:
_broken_ link checker 🙃 ?

Reply:
FYI, there is an interesting dockerfile in autotests, that seems to install every tool on earth for link checking (the job might use some of them): 

- https://github.com/exonum/autotests-exonum-java-binding/blob/master/images/links-verify/Dockerfile 
- http://192.168.221.185:8888/view/EJB/job/exonum-java-binding-links-verify/

Reply:
I believe the name should be interpreted as "(broken link) checker", not "broken (link checker)" 🙃